app/cli/main.go

```diff
@@ -31,9 +31,12 @@ func main() {
 	logger := zerolog.New(zerolog.ConsoleWriter{Out: os.Stderr, FormatTimestamp: func(interface{}) string { return "" }})
 	rootCmd := cmd.NewRootCmd(logger)
 
+	// Run the command
 	if err := rootCmd.Execute(); err != nil {
+		var msg string
```

Comment:
we can probably revert all the changes in this file. For example, having the msg var  doesn't seem to make sense anymore.